doc/configcommands.dsv

```diff
@@ -17,7 +17,7 @@ confirm-exit||[yes/no]||no||If set to `yes`, then Newsboat will ask for confirma
 cookie-cache||<path>||""||Set a cookie cache. If set, cookies will be cached in (i.e. read from and written to) this file, using http://www.cookiecentral.com/faq/#3.5[Netscape format].||cookie-cache "~/.newsboat/cookies.txt"
 datetime-format||<date/time format>||%b %d||This format specifies the date/time format in the article list. For a detailed documentation on the allowed formats, consult the manpage of strftime(3).||datetime-format "%D, %R"
 define-filter||<name> <filterexpr>||n/a||With this command, you can predefine filters, which you can later select from a list, and which are then applied after selection. This is especially useful for filters that you need often and you don't want to enter them every time you need them.||define-filter "all feeds with 'fun' tag" "tags # \"fun\""
-delete-read-articles-on-quit||[yes/no]||no||If set to `yes`, then all read articles will be deleted when you quit Newsboat.||delete-read-articles-on-quit yes
+delete-read-articles-on-quit||[yes/no]||no||If set to `yes`, all read articles will be deleted when you start Newsboat with the `--cleanup` option or quit Newsboat if `cleanup-on-quit` is set.||delete-read-articles-on-quit yes
```

Comment:
That's a very complicated sentence, it's hard to parse correctly even if one is familiar with the code :) I'd split it in two, something like this:

> If set to `yes`, all read articles will be deleted when you run Newsboat with `--cleanup` option, or when you quit it normally. This setting only applies when `cleanup-on-quit` is set.

This is still quite complicated, so I'd be glad if you can improve it further.

Reply:
> This setting only applies when cleanup-on-quit is set.

That sounds incorrect because the setting also applies when only specifying `--cleanup` (even when `cleanup-on-quit` would be `no`).

Do you think the following is clear enough?
> If set to `yes`, all read articles will be deleted when quiting Newsboat. This option only applies if `cleanup-on-quit` is set to `yes` or if the `--cleanup` argument is passed.

Reply:
> That sounds incorrect because …

Ah, yes, you're right! Your new suggestion does sound much clearer, let's go with it.